src/Tribe/Editor/Configuration.php

```diff
@@ -39,15 +83,14 @@ public function editor_config( $editor_config ) {
 	public function localize() {
 		/** @var Tribe__Events__Admin__Event_Meta_Box $events_meta_box */
 		$events_meta_box = tribe( 'tec.admin.event-meta-box' );
-		return array(
+		$data = [
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 12 spaces but found 1 space

<sub>hash: 56cbbb89b480fe857629dc0cce1fdcae</sub>